addons/sys_intercom/XEH_postInit.sqf

```diff
@@ -3,14 +3,76 @@
 // Exit if ACE3 not loaded
 if (!isClass (configFile >> "CfgPatches" >> "ace_interact_menu")) exitWith {};
 
-["Tank", "init", FUNC(intercomConfig), nil, nil, true] call CBA_fnc_addClassEventHandler;
-["Car_F", "init", FUNC(intercomConfig), nil, nil, true] call CBA_fnc_addClassEventHandler;
-["Air", "init", FUNC(intercomConfig), nil, nil, true] call CBA_fnc_addClassEventHandler;
-["Boat_F", "init", FUNC(intercomConfig), nil, nil, true] call CBA_fnc_addClassEventHandler;
+["Tank", "init", FUNC(initVehicleIntercom), nil, nil, true] call CBA_fnc_addClassEventHandler;
+["Car_F", "init", FUNC(initVehicleIntercom), nil, nil, true] call CBA_fnc_addClassEventHandler;
+["Air", "init", FUNC(initVehicleIntercom), nil, nil, true] call CBA_fnc_addClassEventHandler;
+["Boat_F", "init", FUNC(initVehicleIntercom), nil, nil, true] call CBA_fnc_addClassEventHandler;
 
 if (!hasInterface) exitWith {};
 
-ADDPFH(DFUNC(vehicleCrewPFH), 1.1, []);
+["vehicle", {
+    params ["_player", "_newVehicle"];
+
+    if (_player != _newVehicle) then {
+        _player setVariable[QGVAR(intercomVehicle), _newVehicle];
+
+        // Configure the array for handling if a player is in a limited intercom position or not.
```

Comment:
Remove `or not.` (also what do you people have with final dots in comments).

Reply:
What do people have with  sentences without final dots?

Reply:
We don't classify in-line comments as sentences. 😆